<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDiagFlt</class>
 <widget class="QWidget" name="FrmDiagFlt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>54</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="2,2,2,2,2,2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Section</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Diagnostic</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Item</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="cboxSection"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cboxDiagnostic"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="cboxOutput"/>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="cboxItem"/>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="cboxComparison">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(220, 225, 255);
selection-background-color: rgb(0, 255, 217);</string>
         </property>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>≤</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>≠</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>≥</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>contains</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>! contains</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>begins with</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ends with</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>matches</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="edValue"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>4</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tbtnRemove">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="configtool.qrc">
           <normaloff>:/img/img/minus.png</normaloff>:/img/img/minus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbtnAdd">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="configtool.qrc">
           <normaloff>:/img/img/plus.png</normaloff>:/img/img/plus.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="configtool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cboxSection</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmDiagFlt</receiver>
   <slot>changedSection(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboxDiagnostic</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmDiagFlt</receiver>
   <slot>changedDiagnostic(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboxOutput</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmDiagFlt</receiver>
   <slot>changedOutput(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboxItem</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmDiagFlt</receiver>
   <slot>changedItem(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboxComparison</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmDiagFlt</receiver>
   <slot>changedComparison(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edValue</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FrmDiagFlt</receiver>
   <slot>changedCompareTo(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbtnRemove</sender>
   <signal>clicked()</signal>
   <receiver>FrmDiagFlt</receiver>
   <slot>emitRemoveFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbtnAdd</sender>
   <signal>clicked()</signal>
   <receiver>FrmDiagFlt</receiver>
   <slot>emitAddFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>rqstAddFilter()</signal>
  <signal>rqstRemoveFilter()</signal>
  <slot>updateFilter()</slot>
  <slot>changedSection(int)</slot>
  <slot>changedDiagnostic(int)</slot>
  <slot>changedOutput(int)</slot>
  <slot>changedItem(int)</slot>
  <slot>changedComparison(int)</slot>
  <slot>changedCompareTo(QString)</slot>
  <slot>emitAddFilter()</slot>
  <slot>emitRemoveFilter()</slot>
 </slots>
</ui>
